tests/server/index.js

```diff
@@ -1,24 +1,125 @@
+const mongoose = require('mongoose');
 const tape = require('tape');
 const supertest = require('supertest');
 const exphbs = require('express-handlebars');
+require('env2')('./config.env');
 
+mongoose.connect(process.env.DB_URL, {
+  useMongoClient: true
+});
+
+const { mockCollection } = require('../../database/startup_schema.js');
 const server = require('../../src/server.js');
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => {
+  mockCollection.remove({startupName: 'FAC'}, (err) => {
+    if (err) {
+      console.log(err);
+      return
+    }
+    console.log('removed');
+  });
+  console.log("connected db");
+})
+
 
 tape('Test home route', (t) => {
```

Comment:
but im not just testing that im also testing there is no error and the right status